src/bp-messages/classes/class-bp-messages-message.php

```diff
@@ -227,11 +227,25 @@ public function send() {
 	 * @return object $value List of recipients for a message.
 	 */
 	public function get_recipients() {
-		global $wpdb;
 
-		$bp = buddypress();
+		$results = BP_Messages_Thread::get(
+			array(
+				'per_page'        => - 1,
```

Comment:
@janakkaneriya007 why we need to pass `per_page => -1`

Reply:
@dipskakadiya old query didn't have any pagination values in it so we can't limit the results to 20 which is by default.